charts/velero/ci/test-values.yaml

```diff
@@ -1,5 +1,3 @@
-installCRDs: false
```

Comment:
In helm3, there is a flag `--skip-crds` to skips install CRDs.
In helm2, it doesn't support this flag, should we keep the `installCRDs` to let user decides whether to install the CRDs or not?

Reply:
I'd say we should keep it. Doesn't seem it will cause any problem with v3 and it's the only way to set that option with v2.

Reply:
okay, revert it.